lib/remi/data_subjects/gsheet_file.rb

```diff
@@ -0,0 +1,163 @@
+
+module Remi
+
+  require 'google/apis/sheets_v4'
+  require 'google/apis/drive_v3'
+  require 'googleauth'
+  require 'googleauth/stores/file_token_store'
+  require 'daru'
+  require 'fileutils'
+
+  # Contains methods shared between Salesforce Extractor/Parser/Encoder/Loader
+  class Extractor::Gsheet < Extractor::FileSystem
+
+    def initialize(*args, **kargs, &block)
+      super
+      init_gsheet_extractor(*args, **kargs)
+    end
+
+    attr_reader :folder_name
+    attr_reader :itk_file
```

Comment:
`itk_file` isn't used.

Also, Remi is intended to be a public library and shouldn't have any ITK-specific stuff in it.